package.json

```diff
@@ -21,7 +21,7 @@
     "typecheck:web": "tsc --noEmit -p tsconfig.web.json --composite false",
     "typecheck": "npm run typecheck:node && npm run typecheck:web",
     "start": "electron-vite preview",
-    "dev": "electron-vite dev",
+    "dev": "electron-vite dev --noSandbox",
```

Comment:
Devtools doesn't work with electron-vite on some systems (including mine) without being on the host (rather than a sandbox).